tox.ini

```diff
@@ -54,7 +54,6 @@ deps =
     -rrequirements/edx/development.txt
     -rrequirements/edx/testing.txt
     -rrequirements/edx/post.txt
-    -rrequirements/edx/edx-private.txt
```

Comment:
There's a point about why in the PR description. Basically the mentioned test (and 1-2 others) complained about the following XBlocks not being in the graded list:
```
'pb-column', 'sb-review-step', 'step-builder', 'pb-mrq', 'vectordraw',
'activetable', 'ubcpi', 'pb-table', 'pb-mcq', 'problem-builder', 'pb-rating',
'schoolyourself_review', 'sb-plot', 'sb-step'
```
The above blocks are mostly ProblemBuilder XBlocks. Here's the actual test failure:
```
_____________________________________________________________ TestScoredBlockTypes.test_block_types_possibly_scored ______________________________________________________________

self = <lms.djangoapps.grades.tests.test_scores.TestScoredBlockTypes testMethod=test_block_types_possibly_scored>

    def test_block_types_possibly_scored(self):
        self.assertSetEqual(
            self.possibly_scored_block_types,
>           scores._block_types_possibly_scored()
        )
E       AssertionError: Items in the second set but not the first:
E       'pb-column'
E       'sb-review-step'
E       'step-builder'
E       'pb-mrq'
E       'vectordraw'
E       'activetable'
E       'ubcpi'
E       'pb-table'
E       'pb-mcq'
E       'problem-builder'
E       'pb-rating'
E       'schoolyourself_review'
E       'sb-plot'
E       'sb-step'

lms/djangoapps/grades/tests/test_scores.py:65: AssertionError
```
Interestingly, this test fails always on Docker devstack currently due to edx-private.txt's packages being pip-installed there. The Jenkins CI images must *not* have edx-private.txt requirements installed, since this tests passes fine there.

What we really need is consistency across the three testing environments: Docker devstack, Jenkins CI, tox-based Django 1.9+ tests.

Reply:
Oh - and we have no tests that use the edx-private.txt requirements, shown by the fact that all tests pass without those requirements being installed.